pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -108,38 +191,60 @@ func (r *remoteRuntimeService) RunPodSandbox(config *runtimeapi.PodSandboxConfig
 	ctx, cancel := getContextWithTimeout(timeout)
 	defer cancel()
 
-	resp, err := r.runtimeClient.RunPodSandbox(ctx, &runtimeapi.RunPodSandboxRequest{
-		Config:         config,
-		RuntimeHandler: runtimeHandler,
-	})
-	if err != nil {
-		klog.ErrorS(err, "RunPodSandbox from runtime service failed")
-		return "", err
+	var podSandboxID string
```

Comment:
var for the resp and err values..

then after the if else blocks you can do a common if err != nil log and return block then you can just use resp.PodSandboxId